### PR TITLE
Changed commands field to a dictionary, our problems are over.

### LINKED ISSUE #43: A lot of commands in Commands.config.xml cause server lags for any command
Report from RocketMod user (not mine). I can provide a problematic plugin, just let me know.

For some reason when many commands are registered (250+) (commands of the plugin are registered via rocketmod automatically, not manually), then just execute some random like /11 (which doesn't even exist) this causes the server to lag for a few seconds, however, when a plugin with many commands is removed from the Plugin directory (as I understand unloaded) the problem disappears

### PR DESCRIPTION
**Pull request by Jdance-Media**
Made changes that prevent the module from taking a few seconds to execute a command when searching for it.

Resolves #43

### CODE REVIEW

**Comment by Jdance-Media** [comment]
That build failure was not me.

File: Rocket/Rocket.Core/Commands/RocketCommandManager.cs [hunk 1]
```diff
@@ -80,9 +81,9 @@ private IRocketCommand GetCommand(IRocketCommand command)
 
         public IRocketCommand GetCommand(string command)
```
Comment (BlueBeard63) [inline Rocket/Rocket.Core/Commands/RocketCommandManager.cs:82]: Would it not be possible to return a bool and out a IRocketCommand, then you dont have to check for null and just see if its a true or false return.

Comment (Jdance-Media) [reply Rocket/Rocket.Core/Commands/RocketCommandManager.cs:82]: Not the scope of this PR.

Comment (BlueBeard63) [reply Rocket/Rocket.Core/Commands/RocketCommandManager.cs:82]: Ik i was just pointing it out.

**Review by BlueBeard63 — APPROVED** [review]
Code is clean and well done, recommend merge.

**Pull request merged**